scripts/obtain/dotnet-install.sh

```diff
@@ -56,6 +56,59 @@ say_verbose() {
     fi
 }
 
+get_os_download_name_from_platform() {
+    eval $invocation
+
+    platform="$1"
+    case "$platform" in
+        "centos.7")
+            echo "centos"
+            return 0
+            ;;
+        "debian.8")
+            echo "debian"
+            return 0
+            ;;
+        "fedora.23")
+            echo "fedora.23"
+            return 0
+            ;;
+        "fedora.24")
```

Comment:
Can we do a default where if it does not match any of the cases it simply returns the input value? so that we can remove cases like fedora.24, return fedora.24?